main.js

```diff
@@ -15,3 +111,81 @@ const initApp = () => {
 document.addEventListener('DOMContentLoaded', () => {
   initApp();
 });
+
+// model
+
+const modelSection = document.querySelector('.model');
+const modelCloseButton = document.querySelector('.model-close');
+const overlay = document.getElementById('overlay');
+const projectButtons = document.querySelectorAll('.project-btn');
+
+function showPopup() {
+  modelSection.style.display = 'block';
+  overlay.style.display = 'block';
+}
+
+function hidePopup() {
+  modelSection.style.display = 'none';
+  overlay.style.display = 'none';
+}
+
+// document.querySelector(".btn-live").addEventListener("click", showPopup);
+modelCloseButton.addEventListener('click', hidePopup);
+
+projectButtons.forEach((button) => {
+  button.addEventListener('click', showPopup);
+});
+
+// dynamic modal
+const dynamicSection = document.getElementById('dynamic-section');
+
+const sectioHTML = `
+<section id="model-section" class="model-section">
+      <div class="model">
+        <div class="model-content">
+          <div class="model-header">
+            <button class="model-close" type="button">X</button>
+            <img src="./images/Snapshoot Portfolio (1).png" alt="Model-image" />
+          </div>
+          <div class="model-body">
+          <div class="header-model">
+            <h2>Keeping track of hundreds of components</h2>
+            <div class="mobile-list">
+              <ul id="mobile-components"></ul>
+            </div>
+            <div class="desktop-list">
+              <ul id="desktop-componets"></ul>
+            </div>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s.
+            </p>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s.
+            </p>
+            </div>
+          <div class="model-footer desktop-flex">
+            <button class="btn-live" id="live-button">
+              See live
+              <span
+                ><i
+                  class="fa-thin fa-arrow-up-left-from-circle fa-rotate-90"
+                ></i
+              ></span>
+            </button>
+            <button class="btn-source" id="source-button">
```

Comment:
- [ ] Please as stated earlier, kindly ensure that this is generated dynamically  with JavaScript. This portion of code is still html hardcoded in js file. I think you can add other details of pop-up to your `const project` data and populate it the way you did for modal container. 👍

Reply:
- [ ] Awesome designing the pop-up according to the design, but kindly remove the vertical scroll as suggested earlier for good user experience. **Kindly take not that the same is applicable to the mobile version as well.** 👍 

##  Your Design

<img width="959" alt="Untitled" src="https://github.com/Umuhan/My-Portfolio-Project/assets/103386183/d5e4ebe7-20f5-48b8-b514-56fcf961dd2e">

Reply:
done